saos-webapp/src/test/java/pl/edu/icm/saos/webapp/judgment/detail/correction/JudgmentCorrectionServiceTest.java

```diff
@@ -0,0 +1,194 @@
+import pl.edu.icm.saos.persistence.correction.model.CorrectedProperty;
+import pl.edu.icm.saos.persistence.correction.model.JudgmentCorrection;
+import pl.edu.icm.saos.persistence.correction.model.JudgmentCorrectionBuilder;
+import pl.edu.icm.saos.persistence.model.CommonCourtJudgment;
+import pl.edu.icm.saos.persistence.model.Judge;
+import pl.edu.icm.saos.persistence.model.Judgment;
+import pl.edu.icm.saos.persistence.model.SupremeCourtChamber;
+import pl.edu.icm.saos.persistence.model.SupremeCourtJudgment;
+import pl.edu.icm.saos.persistence.model.SupremeCourtJudgmentForm;
+
+import com.google.common.collect.Lists;
+
+@RunWith(MockitoJUnitRunner.class) 
+public class JudgmentCorrectionServiceTest {
+
+	
+	@Mock
+	private JudgmentCorrectionRepository judgmentCorrectionRepository;
+	
+	private JudgmentCorrectionService judgmentCorrectionService = new JudgmentCorrectionService();
+	
+	private Judgment ccJudgment = new CommonCourtJudgment();
+	private Judgment scJudgment = new SupremeCourtJudgment();
+	
+	
+	//------------------------ TESTS --------------------------
+	
+	@Test
+	public void findByJugmentId_CommonCourtJudgment() {
+		//given
+		int judgmentId = 123;
+		
+		List<JudgmentCorrection> unsortedJc = Lists.newArrayList();
+		unsortedJc.add(createJcJudgeDelete(ccJudgment));
+		unsortedJc.add(createJcCommonCourtJudgment());
+		
+		
+		List<JudgmentCorrection> expectedJc = Lists.newArrayList();
+		expectedJc.add(createJcCommonCourtJudgment());
+		expectedJc.add(createJcJudgeDelete(ccJudgment));
+		
+		when(judgmentCorrectionRepository.findAllByJudgmentId(judgmentId)).thenReturn(unsortedJc);
+		judgmentCorrectionService.setJudgmentCorrectionRepository(judgmentCorrectionRepository);
+		
+		
+		//execution
+		List<JudgmentCorrection> actualJc = judgmentCorrectionService.findByJugmentId(judgmentId); 
+		
+		
+		//assertion
+		assertEquals(expectedJc, actualJc);
+	}
+	
+	@Test
+	public void findByJugmentId_SupremeCourtJudgment() {
+		
+		//given
+		int judgmentId = 124;
+		
+		List<JudgmentCorrection> unsortedJc = Lists.newArrayList();
+		unsortedJc.add(createJcSupremeCourtChamber());
+		unsortedJc.add(createJcJudgeDelete(scJudgment));
+		unsortedJc.add(createJcSupremeCourtJudgmentForm());
+		unsortedJc.add(createJcSupremeCourtJudgment());
+		
+		List<JudgmentCorrection> expectedJc = Lists.newArrayList();
+		expectedJc.add(createJcSupremeCourtJudgment());
+		expectedJc.add(createJcSupremeCourtChamber());
+		expectedJc.add(createJcSupremeCourtJudgmentForm());
+		expectedJc.add(createJcJudgeDelete(scJudgment));
+		
+		
+		when(judgmentCorrectionRepository.findAllByJudgmentId(judgmentId)).thenReturn(unsortedJc);
+		judgmentCorrectionService.setJudgmentCorrectionRepository(judgmentCorrectionRepository);
+		
+		
+		//execution
+		List<JudgmentCorrection> actualJc = judgmentCorrectionService.findByJugmentId(judgmentId); 
+		
+		
+		//assertion
+		assertEquals(expectedJc, actualJc);
+	}
+
+	@Test
+	public void findByJudgmentId_Judges() {
```

Comment:
I wonder if this method is necessary, it's enough to extend the tests for judgments above.